nifi-nar-bundles/nifi-accumulo-bundle/nifi-accumulo-processors/src/main/java/org/apache/nifi/accumulo/processors/ScanAccumulo.java

```diff
@@ -270,8 +279,9 @@ public void process(final InputStream in, final OutputStream out) throws IOExcep
                                         data.put("columnQualifier", key.getColumnQualifier().toString());
                                         data.put("columnVisibility", key.getColumnVisibility().toString());
                                         data.put("timestamp", key.getTimestamp());
+                                        data.put("value", kv.getValue());
```

Comment:
The schema here indicates that this field is to be a `String` type. That means that the value added into the map needs to be a String also, but it's of type `Value`. Need to ensure that it's properly converted to a `String` before putting into the Map.

Reply:
Thanks Mark for your help, modified to kv.getValue().toString()